src/test/java/com/trello/authorization/AuthorisationTest.java

```diff
@@ -0,0 +1,53 @@
+package com.trello.authorization;
+
+import com.trello.BoardPage;
+import com.trello.LogInPage;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class AuthorisationTest {
+
+    private ChromeDriver driver;
+    LogInPage logInPage = new LogInPage(driver);
```

Comment:
Do not crate page object as test field. Use separate pages in each test

Reply:
Done, if I understand right.